azure-pipelines.yml

```diff
@@ -123,21 +123,15 @@ stages:
         variables:
         # Enable signing for internal, non-PR builds
         - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
-          - group: DotNet-Blob-Feed
           - group: DotNet-Symbol-Server-Pats
           - name: _SignType
             value: Real
-          - name: _DotNetPublishToBlobFeed
-            value: true
           - name: _BuildArgs
             value: /p:SignType=$(_SignType)
                    /p:DotNetSignType=$(_SignType)
                    /p:MicroBuild_SigningEnabled=true
                    /p:OverridePackageSource=https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json
                    /p:TeamName=$(_TeamName)
-                   /p:DotNetPublishBlobFeedKey=$(dotnetfeed-storage-access-key-1)
-                   /p:DotNetPublishBlobFeedUrl=https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json
-                   /p:DotNetPublishToBlobFeed=true
```

Comment:
Curious what was this blob feed being used for?

Reply:
It used to be a method of publishing NuGet packages pre-AzDO feed days. We have long since removed support for it. I think you can also probably remove OverridePackageSource (but confirm its usage within your repo)